src/templates/settings/export/png.html

```diff
@@ -33,6 +34,16 @@
       </div>
     </div>
 
+    <div class="export-panel-section">
```

Comment:
For me it doesn't fit if I have more than one frame in my sprite. When there is more than 1 frame there is an extra "layout section" that is introduced in the `PNG` tab.

Since this doesn't really care about the layout options, we could move it to the `Other` tab?

Reply:
Hmmm actually it does use the layout options.

Reply:
So should it go in the 'other' tab or should I bump up the height of that drawer?

Reply:
With a multi-frame sprite I can get the drawer to (just) fit without any overlap on my pretty small laptop screen... but it's quite snug.

Initially, I had the CSS export as a checkbox option with the sprite sheet export which was slightly less height intensive, so that's maybe another option.

Let me know where you want it and I'll adjust and/or move it.